SofaKernel/modules/CMakeLists.txt

```diff
@@ -2,12 +2,16 @@
 # need to first add find_package(XXX) in your CMakeLists.txt
 
 # Common
-sofa_add_module(SofaSimpleFem SofaSimpleFem ON)
-sofa_add_module(SofaRigid SofaRigid ON)
-sofa_add_module(SofaDeformable SofaDeformable ON)
-sofa_add_module(SofaObjectInteraction SofaObjectInteraction ON)
-sofa_add_module(SofaMeshCollision SofaMeshCollision ON)
-sofa_add_module(SofaEngine SofaEngine ON)
-sofa_add_module(SofaExplicitOdeSolver SofaExplicitOdeSolver ON)
-sofa_add_module(SofaImplicitOdeSolver SofaImplicitOdeSolver ON)
-sofa_add_module(SofaLoader SofaLoader ON)
+sofa_add_plugin(SofaSimpleFem SofaSimpleFem ON)
+sofa_add_plugin(SofaRigid SofaRigid ON)
+sofa_add_plugin(SofaDeformable SofaDeformable ON)
+sofa_add_plugin(SofaObjectInteraction SofaObjectInteraction ON)
+sofa_add_plugin(SofaMeshCollision SofaMeshCollision ON)
+sofa_add_plugin(SofaEngine SofaEngine ON)
+sofa_add_plugin(SofaExplicitOdeSolver SofaExplicitOdeSolver ON)
+sofa_add_plugin(SofaImplicitOdeSolver SofaImplicitOdeSolver ON)
+sofa_add_plugin(SofaLoader SofaLoader ON)
+
+sofa_add_module(Sofa.GL Sofa.GL ON
```

Comment:
It was in the previous PR but Sofa.GL should be at the top of this file, as SofaSimpleFem, SofaRigid, etc _could need_ Sofa.GL ; in this current setup they will never find it.

Reply:
Ok I update this :+1: